pkg/controller/testutil/test_utils.go

```diff
@@ -313,31 +313,31 @@ func (m *FakeNodeHandler) Patch(_ context.Context, name string, pt types.PatchTy
 	case types.JSONPatchType:
 		patchObj, err := jsonpatch.DecodePatch(data)
 		if err != nil {
-			klog.Error(err.Error())
+			klog.ErrorS(err, "Error")
```

Comment:
This would just be an empty string as there are no arguments
```suggestion
			klog.ErrorS(err, "")
```

Reply:
Fixed.